python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -98,13 +122,23 @@ def _computePsfImage(self, position=None):
     if not isinstance(position, Point2D) and position is not None:
         position = Point2D(position[0], position[1])
 
-    for single in self.singles:
-        if position is None:
-            psf = single.getPsf().computeImage()
-            psfs.append(psf)
-        else:
-            psf = single.getPsf().computeImage(position)
-            psfs.append(psf)
+    for band, single in enumerate(self.singles):
```

Comment:
I recommend consistently naming iteration indices idx_ or something that indicates that it's an index/integer since band is often a string.